semantic-python/test/fixtures/2-01-return-statement.py

```diff
@@ -1,3 +1,3 @@
-# CHECK-JQ: .tree.contents[0][1].contents[1].contents.value == []
+# CHECK-JQ: .tree.contents[0][1].contents[1].contents.value.value == []
```

Comment:
These JSON trees are starting to get pretty ridiculous. We need to tackle #245 sooner rather than later.